draft-ietf-tls-esni.md

```diff
@@ -456,6 +456,9 @@ The client then constructs the ClientHelloInner message just as it does a
 standard ClientHello, with the exception of the following rules:
 
 1. It MUST NOT offer to negotiate TLS 1.2 or below.
+1. It MUST ensure that all HelloRetryRequest-sensitive parameters applicable
+   to TLS 1.3 and higher versions in ClientHelloInner match that in
+   ClientHelloOuter. See {{hrr}} for the list of these parameters.
 1. It MUST NOT offer to resume any non-ECH PSK or any session for TLS 1.2 and
```

Comment:
"non-ECH PSK": does this term still exist? I know we talked about this already but I forgot where we landed :)

Reply:
It's addressed in another PR.